README.md

```diff
@@ -34,7 +34,7 @@
 Pandera is a [Union.ai](https://union.ai/blog-post/pandera-joins-union-ai) open
 source project that provides a flexible and expressive API for performing data
 validation on dataframe-like objects. The goal of Pandera is to make data
-processing pipelines more readable and robust with statistically typed
```

Comment:
I feel this is likely intentional, given how often "statistical" is written in https://proceedings.scipy.org/articles/Majora-342d178e-010.pdf, but I'll let @cosmicBboy weigh in before closing it.

Reply:
Oh I see! I imagine you're right. Sorry for the noise. Please do feel free to close this.